app/lib/shared/configuration.dart

```diff
@@ -384,6 +306,7 @@ class EnvConfig {
       Platform.environment['TOOL_PREVIEW_FLUTTER_SDK'],
       frontendCount,
       workerCount,
+      Platform.environment['PUB_CONFIG'],
```

Comment:
Let's call this `PUB_SITE_CONFIG` or `PUB_SERVER_CONFIG` in case we may have a similar config for the `pub` command line tool.